# 1. Introduction  

Early life stress, especially if chronic, is related to negative behavioral, cognitive and health outcomes. Entering center-based childcare (CC) at three months of life, the normal age in the Netherlands, is an important change or ‘life event’ for an infant. CC includes stressors such a long maternal separations, new and frequently changing caregivers and peers, and adaptation to a new physical environment. From our own and other studies, we know that entering childcare produces significant increases in cortisol levels as compared to levels in the home environment, and that cortisol continues to increase until at least a month after entering [@Ahnert2004; @Albers2016; @Bernard2015; @Watamura2010; @Waynforth2007]. CC has also been related to more health symptoms and illnesses including diarrhea and communicable diseases such as respiratory illnesses and otitis media [@Beijers2011; @Loeb2007].

A growing body of evidence indicates that the intestinal microbiota might play an important role in how such early life stressors influence future health outcomes and behavior. Firstly, in human studies gut microbiota composition is associated with several pediatric disorders including allergic and autoimmune diseases [@Videhult2016]. For example, @Azad2015 found a lower richness and increased _Enterobacteriaceae/Bacteroidetes_ ratio in infant stool to be associated with subsequent food sensitization in the population based Canadian Healthy Infant Longitudinal Development (CHILD) birth cohort study. Secondly, animal studies indicate that stress affects the gut microbiota, which in turn leads to inflammatory immune processes and even behavioral changes. For example, rodent studies showed that early life stress induced by maternal separation led to alternations of the intestinal barrier function and the microbiota [@Rea2016]. These alternations were accompanied by behavioral changes that in one study have been found to be mediated by the intestinal bacteria [@DePalma2015]. Interestingly, the administration of probiotics could prevent or ameliorate stress induced effects [@Rea2016]. In rhesus monkeys, maternal separation also altered the composition of the intestinal microbiota of the offspring by decreasing the number _Lactobacilli_ [@Bailey1999].  However, we lack studies that investigated the effect of early life stress on the development of the gut microbiota composition in humans and specifically the effect of CC on the infant gut microbiota composition has not been studied at all.  
Among the better studied factors that influence the development of the infant gut microbiota are diet and specifically breastfeeding. Breastfeeding increases the abundance of Lactobacilli and Bifidobacteria species when compared to formula fed infants [@Gregory2016; @Videhult2016]. @Yamada2017 described how oligosaccharides present in human breast milk selectively stimulate the growth of Bifidobacteria. Finally, there is evidence suggesting that breast milk contains bacteria that colonize the infant gut. For instance, @Pannaraj2017 describe that on average, 27.7% of the infant gut microbiome resembles the microbiome of the mother’s breast milk and 10.3% resembles the microbiome of the mother’s areolar skin. In contrast to early life stress, breastfeeding is known to be a health promoting factor in early life [@Hoddinott2008].  
Altogether, this research suggests that early life stress might impair the development of the intestinal microbiota with possible long term effects on physical health and behavior and that breastfeeding might be a protective factor against the negative influence of stress. In this study we investigated whether the entrance to CC leads to changes in the intestinal microbiota and whether breastfeeding plays a role in this relationship. We obtained stool samples from infants who entered CC (*n*=49) and control infants (*n*=49) before the entrance at 10 weeks of age (T0) and four weeks after the entrance (T1). We hypothesized (1) that we find no differences in the composition of the intestinal microbiota between infants who entered CC and control infants at T0 but that we find differences at at T1. The effect of CC might be moderated by a buffering effect of breastfeeding on microbiota changes. Therefore, (2) we anticipate a smaller effect on microbiota composition in the breastfed group compared to the non breastfed group among infants that attended CC. 
   

# 2. Methods
## 2.1 Participants

As part of an ongoing longitudinal study (Bibo study), 220 mothers were followed since the third trimester of pregnancy, to investigate the influences of early caregiving factors on the early development of children. Uncomplicated singleton pregnancy, proficiency in the Dutch language, no drug use and the absence of physical and mental health problems were criteria for initial inclusion. Eight of the 220 women were excluded due to preterm birth or for other medical reasons. In addition, 19 mothers discontinued participation in the study during the first three postpartum months because of personal circumstances. All infants were healthy and born at full term ($\geq$ 37 weeks). Infants born by cesarean delivery (n=4) and who had used antibiotics (n=4) were excluded. Out of a total of 9 fecal samples that were available per infant, 2 were selected for this study: Ten weeks post-partum before entrance to CC (T0) and 4 weeks after the entrance (T1). If there was no sample obtained at both times points, the infant was excluded. Seven samples were excluded for technical reasons. For these samples, the Pearson correlation of $\>.98$ between duplicate bacterial profiles was not reached. The final sample included 49 infants per group. The study has been approved by the ethical Committee of the Faculty of Social Sciences, Radboud University Nijmegen (ECG / AvdK/07.563) and informed consent was obtained from each mother.



##  2.2 Procedure

### 2.2.1 Breastfeeding
Information about breastfeeding  was collected through the use of diaries. The mothers received diaries towards the end of their pregnancy with instructions to take weekly notes about breastfeeding and bottle-feeding from week 1 until week 27 after birth. For each week, the average number of breast- and/or bottle- feedings per day were noted. Infants who did not receive breastfeeding between T0 and T1 were categorized as not breastfed whereas infants who have been breastfed more than 50% of the time between T0 and T1 were categorized as breastfed. The number of weeks infants have been breastfed before T0 also were taken into account in the analyses.

### 2.2.2 Faeces collection, DNA isolation and microbiota profiling
The parents were instructed to collect the fecal samples at home and to store them at -20°C. Samples were kept in coolers for transportation and then stored at -20°C and later at -80°C before being processed at the Microbiology Laboratory at Wageningen University. DNA isolation from faecal samples has been described in detail elsewhere [@Salonen2010]. In brief, DNA was isolated using a combination of column purification and Repeated-Bead-Beating.  Purity and concentration of DNA were assessed with a Nanodrop 1000 spectrophotometer (Thermo Fisher Scientific, Wilmington, USA). DNA was then stored at -20 C° pending microbial analysis. The analysis was then performed utilizing a previously benchmarked custom made, phylogenetic microarray, the Human Intestinal Tract Chip (HITChip) [@Rajilic-Stojanovic2009; @Claesson2009a]. The HITchip contains a duplicated set of 3,631 probes, which target the 16S rRNA gene sequences of more than 1,000 intestinal bacterial phylotypes [@Rajilic-Stojanovic2009]. After extraction of DNA, the full-length 16S rRNA gene was amplified, which was followed by in vitro transcription and labelling of the resulting RNA with Cy3/Cy5 before hybridization to the array. Each sample was hybridized at least twice to ensure reproducibility. Duplicate hybridizations with a Pearson correlation &lt;.98 were not considered for further analysis. By pre-processing the probe-level measurements with minimum–maximum normalization and Robust Probabilistic Averaging probe summarization  [@Lahti2011; @Lahti2013] the microbiota profiles were generated  into three phylogenetic levels: Order-like 16S rRNA gene sequence groups (level 1); genus-like 16S rRNA gene sequence groups with a sequence similarity &gt;90% (level 2); and phylotype-like 16S rRNA gene sequence groups with a sequence similarity &gt;98% (level 3). For our study, we focus on the genus-level variation referred to as species and relatives (‘et rel.’) and use log10-transformed signals as a proxy for bacterial abundance. 


### 2.2.3 Additional variables
Maternal age as well as infant birthweight, age in weeks at the entrance to CC and sex were obtained as potential confounding variables. Table 1 shows descriptive statistics for the variables we accounted for.

## 2.3 Statistical analysis
The analyses were performed in R version 3.4 [@R2017]. HITchip log10-transformed signals were used as a proxy for bacterial logarithmic abundance. To test our first hypothesis and determine whether and how much entrance to CC affects microbiota overall community composition, we performed a two way permutational multivariate analysis of variance (PERMANOVA) based on Bray-Curtis dissimilarity matrix for both time points separately. The PERMANOVA was performed using the function _adonis_ from the package _vegan_ [@vegan2017] with CC (CC versus no CC) and breastfeeding (yes versus no) as independent variables.  Furthermore, we used  the _lme4_ package [@Bates2015] for linear mixed effects modeling (LME) to assess for each genus level separately whether CC has an effect on abundance. The LME was specified with CC, time (T0 versus T1), breastfeeding (yes versus no) and the covariates listed in table 1 as fixed effects. CC, breasfeeding and time were modeled as three way interaction. The random effects were specified to reach a maximal model according to the recommendations by @Barr2013. To determine p values we computed Type 3 conditional F tests with _Kenward-Roger_ approximation for degrees of freedom since this method produces acceptable Type 1 error rates even for smaller sample sizes according to @Luke2017. The F-test is implemented in the Anova function of the package _car_ [@car2011], which in turn calls the function _KRmodcomp_ of the package _pbkrtest_ [@Halekoh2014]. Benjamini-Hochberg correction [@benjamini1995; @Stephens2016] was applied to account for multiple testing.  


# 3. Results 

## 3.1 Permutational multivariate ANOVA 

In a first step we compared the overall community composition using two separate two-way PERMANOVA. An important assumption for PERMANOVA is multivariate homogeneity of group dispersions (variances) [@Anderson2017]. We used the function _betadisper_ [@vegan2017], which utilizes the PERMDISP2 procedure as implemented by Marti Anderson and found that this assumption was met for the pre-test using the samples collected at T0 (_F_(1,96) = 0.38, _p_ = .54) and the post-test using the samples collected at T1 (_F_(1,96) = 1.59, _p_ = .21). 

### 3.1.1 Effects of CC on microbiota composition
Our first hypothesis would predict no significant differences when we compare the CC groups (CC versus no CC) using only the T0 samples whereas we should observe a significant difference using only the T1 samples. Both PERMANOVAs did not find a significant effect of CC on overall community composition meaning that indeed our samples did not differ significantly between CC groups at T0 (_F_(1,97) = 1.02, _p_ = .39, $R^2$ = .01). However, there was also no significant difference at T1 between CC groups (_F_(1,97) = 1.02, _p_ = .38, $R^2$ = .01) so that we could not reject our first null hypothesis with regard to overall community composition. Figure 1 shows the taxa that contributed most to differences in overall community composition in our model whereas figure 2 shows non-metric multidimensional scaling (NMDS) plots including 95% confidence intervals. NMDS is an ordination technique which visualizes the level of similarity of individual samples based on a distance matrix. Figure 2 illustrates the original position of bacterial communities in multidimensional space as accurately as possible using 2 dimensions, for easier interpretation of compositional structure of the dataset. In this specific case we visualize why the PERMANOVA showed no significant difference in composition between our grouping variable CC. Both groups show no visible separation. 

### 3.1.2 Protective effects of breastfeeding
According to our second hypothesis, we would have expected a significant interaction between breastfeeding and CC, which would have indicated that the effect of CC is significantly different between the two breastfeeding groups. Follow up tests then would have revealed a smaller effect of CC for the breastfed infants. Since we could not find a significant effect of CC on microbiota composition we could not statistically test our second hypothesis that breastfeeding decreases the hypothesized effect of CC. Noteworthy, neither was the interaction term significant, nor did breastfeeding alone have a significant impact on overall community composition. Table 2 shows all model parameters for the PERMANOVAs.


## 3.2 Linear mixed effects models
In the next step, we looked at individual bacterial abundances using LMEs. We evaluated the model assumptions of homogeneity of variance and normality of residuals individually for each of the 130 models by visual inspection of residual- and quantile-quantile plots (see figures 3 and 4 for examples). These figures show that, despite log transformation, the assumption of homogeneity of variance and normality of residuals could not be met. Importantly, most of our models showed at least a mild violation of the homogeneity of variance assumption. Several models violated both model assumptions severely. Therefore, the following results need to be interpreted with caution. 

### 3.2.1 Effects of CC on microbiota composition
Within the LME framework, our first hypothesis would predict a significant interaction between the variable that indicates the two time points (T0 vs T1) and the variable that indicates CC (CC versus no CC) for more bacterial genus abundances than expected by chance. Out of the 130 models, 10 indicated a significantly better fit if the interaction term was included. However, after adjusting for multiple testing based on @benjamini1995, no model remained significant. For exploratory purposes we still looked at the magnitude and direction of the effects for all models that were significant in the first place. The interaction term improved model fit most significantly for the genus _Enterococcus_ (_F_(1,94) = 7.84, _p_ = .006). Table 3 shows the log fold changes for all significant models including test statistics of the F test with Kenward-Roger approximation for degrees of freedom. Figure 5 illustrates at the example for _Enterococcus_ the change in log transformed abundance over time within both groups. We see that, on average, for infants who entered CC the abundance of _Enterococcus_ decreases over 4 weeks whereas it increases over 4 weeks for infants who did not enter CC. However, looking at the individuals reveals that for many infants the opposite is the case. Figures 6-14 show the same phenomenon of highly individual variation for the other species.
  
### 3.2.2 Protective effects of breastfeeding
Our second hypothesis would have predicted a significant three-way interaction between the time-points, CC and breastfeeding with follow up models showing that the effect of CC over time is significantly larger for non-breastfed infants. However, this hypothesis could not be tested due to the absence of an effect of CC over time.


# 4. Discussion 

We utilized both multivariate and univariate statistical approaches to test our hypotheses that CC has an effect on gut microbiota composition . Our analyses did not indicate a significant effect of CC over time and therefore both null hypotheses could not be rejected for our sample. Unexpectedly, breastfeeding and CC together explained only small amounts of variation in overall microbiota composition.  
This result is in contrast to the previously cited rodent studies [@Rea2016], but in line with @Falony2016 who recently explored the drivers of microbiota variation and similarly showed very small effect sizes for all covariates under investigation. In total, 69 of 503 (13.7%) metadata variables were significantly associated with microbiota composition and these had a cumulative (non-redundant) effect size on community variation of only 7.63%. Thus, more than 92% of microbiota variation remained unexplained suggesting additional contribution from unknown factors, such as stochastic effects and biotic interactions.  
In animal experiments the variable under investigation is usually the only factor that is changed, while in human populations the hosts show much more variation with regards to genetics and lifestyle, while the microbiota is subject to much more interactions with the environment [@Falony2016; @Rea2016].  
Nevertheless, the structure, function and stability of the infant microbiota is significantly different in composition as well as diversity from that of adults [@DeMuinck2018]. Especially in the first few weeks the gut microbiota composition is highly unstable and individual. This period is followed up by a more stable phase dominated by _Bifidobacterium_ species when breastfed and gradually goes to a more mature and diverse community during weaning due to the introduction of solid foods. It is yet unclear at what age the microbiota fully matures, but estimates range from 1 to 3 years of age [@DeMuinck2018].
This difference in microbiota dynamics in the first months of life could explain the small effect sizes we observed. Because the microbiota is so unstable it might be that there are no general effects of microbiota composition, in other words: Every infant responds in a unique way. This seems to be the case as can be seen by figures 5-14 where we do observe an average response, but a high percentage of infants shows an opposite effect. Inspecting the same figures for the remaining 120 bacterial groups, we also find highly variable trends between T0 and T1, which explains why we don’t observe an overall general effect. Another line of reasoning could be that, because of the lower stability of the infant microbiota compared to adults, it is therefore more easily ‘pushed’ or shifted due to an environmental influence such as the entrance to CC. This could still be the case. However, our data suggests the effects of this ‘push’ then are highly individual.  
Another observation is that unfortunately the period of the introduction of the first solid foods often coincided with the period under investigation, possibly combining multiple additional variables that affect the microbiota, thus obscuring separate effects. Further research would elucidate whether this is the case and we could then make classes for instance based on the percentage of solid food.  
Contrasting with previous studies we found no significant differences in the abundance of _Bifidobacterium_ between breastfed and non-breastfed infants (see figure 15). This can be explained by the addition of prebiotic fibers in infant formula’s such as Galactooligosaccharides (GOS) and Fructooligosaccharides (FOS) [@Borewicz2018]. These fibers have been added in 2002 to most infant formulae (personal communication) to emulate the bifidogenic effect of breastmilk.  
In addition to biological explanations for our observations, we encountered several methodological limitations. First, only a limited number of infants were not breastfed per group (18 infants who entered childcare and 15 infants who did not enter childcare). Larger sample sizes might be required to find an effect of breastfeeding or interactions between breastfeeding and CC. Secondly, the distributional properties of the taxa could not be addressed by solely using a gaussian model. Despite log transforming the abundances, the assumptions of normality of residuals and heterogeneity of variance often could not be met for our LMEs. We used non-parametric tests for exploratory purposes, which are not affected by the distributional properties but these do not take into account individual variation, which is a strength of LME when testing repeated measurements [@Barr2013]. A possible solution could be to analyze the data taking into consideration the individual distribution for each bacterial genus e.g. by using different link functions for the generalized linear mixed effects models. It should be noted though that for the purpose of inference as opposed to prediction, the assumptions of normality of residuals and homogeneity of variance are less important and at least mild violations should be unproblematic [@Gelman2007].  
In conclusion, based on our analyses it seems that childcare attendance does not influence the microbiota in a large and distinct way, beyond normal temporal variation. This does not mean that childcare attendance does not have an effect on the microbiota, but might imply that its effect on the microbiota are either very individual or are only observable at a different level, such as through bacterial metabolites.




```{r results = 'hide', fig.show = 'hide', message = FALSE, warning = FALSE}
# own functions
source("~/workspace/own/reporting.R")
source("~/workspace/research_master/dpb/minor_research_project/article/analyses/gerben/adj_linear mixed model for Henrik to measure between groups changes.R")
```

```{r results = 'hide', fig.show = 'hide', message = FALSE, warning = FALSE}
source("/Users/henrikeckermann/workspace/research_master/dpb/minor_research_project/article/analyses/tables/apa_tables.R")
library(ggpubr)
```

```{r fig.show=T, fig.cap='Taxa that contribute most to the community differences between childcare groups for timepoints T0 (A) and T1 (B) as reflected by their coefficient size in the PERMANOVA model.'}
ggarrange(top_taxa_pre + ggtitle('A'), top_taxa_post + ggtitle('B'), nrow = 2, common.legend = T)
```

```{r fig.show=T, fig.cap="Non-metric multidimensional scaling plot illustrating differences between childcare groups at timepoints T0 (A) and T1 (B). The ellipses illustrates 95% confidence intervals."}
ggarrange(nmds_pre + ggtitle('A'), nmds_post + ggtitle('B'),common.legend = TRUE)
```

```{r fig.show=T, fig.cap='Density (A) and quantile-quantile plot (B) of standardized residuals, fitted values plotted against standardized residuals (C) and fitted values plotted against observed values (D) for the linear mixed effects model of Enterococcus.'}
ggarrange(resid_den + ggtitle('A'), qq + ggtitle('B'), fit_resid+ ggtitle('C'),fit_obs + ggtitle('D'))
```






```{r fig.show=T, fig.cap='Density (A) and quantile-quantile plot (B) of standardized residuals, fitted values plotted against standardized residuals (C) and fitted values plotted against observed values (D) for the linear mixed effects model of Bacteroides fragilis et rel. Red points correspond to standardized residuals $>3$ (model outlier).'}
ggarrange(resid_den_bac_frag + ggtitle('A'), qq_bac_frag + ggtitle('B'), fit_resid_bac_frag+ ggtitle('C'),fit_obs_bac_frag + ggtitle('D'))
```

```{r fig.show=T, fig.cap='Log fold change in abundance of Enterococcus within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Enterococcus
```


```{r fig.show=T, fig.cap='Log fold change in abundance of Granulicatella within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Granulicatella
```

```{r fig.show=T, fig.cap='Log fold change in abundance of Anaerofustis within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Anaerofustis
```

```{r fig.show=T, fig.cap='Log fold change in abundance of Aerococcus within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Aerococcus
```


```{r fig.show=T, fig.cap='Log fold change in abundance of Aquabacterium within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Aquabacterium
```

```{r fig.show=T, fig.cap='Log fold change in abundance of Bilophila et rel within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Bilophila_et_rel
```

```{r fig.show=T, fig.cap='Log fold change in abundance of Bifidobacterium within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Bifidobacterium
```

```{r fig.show=T, fig.cap='Log fold change in abundance of Aeromonas within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Aeromonas
```

```{r fig.show=T, fig.cap='Log fold change in abundance of Clostridium ramosum et rel within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Clostridium_ramosum_et_rel
```

```{r fig.show=T, fig.cap='Log fold change in abundance of Asteroleplasma et rel within groups of centerbased childcare (CC) or no centerbased childcare (noCC).'}
all.graphs$Asteroleplasma_et_rel
```

```{r fig.show=T, fig.cap='Distribution of Bifidobacteria between breastfed and formula fed infants.'}
bf_dist
```
