---
output:
  word_document: default
  html_document: default
---
# 1. Introduction  

Early life stress, especially if chronic, is related to negative behavioral, cognitive and health outcomes. Entering center-based childcare (CC) at three months of life, the normal age in the Netherlands, is an important change or ‘life event’ for an infant. CC includes stressors such a long maternal separations, new and frequently changing caregivers and peers, and adaptation to a new physical environment. From our own and other studies, we know that entering childcare produces significant increases in cortisol levels as compared to levels in the home environment, and that cortisol continues to increase until at least a month after entering [@Ahnert2004; @Albers2016; @Bernard2015; @Watamura2010; @Waynforth2007]. CC has also been related to more health symptoms and illnesses including diarrhea and communicable diseases such as respiratory illnesses and otitis media [@Beijers2011; @Loeb2007].

A growing body of evidence indicates that the intestinal microbiota might play an important role in how such early life stressors influence future health outcomes and behavior. Firstly, in human studies gut microbiota composition is associated with several pediatric disorders including allergic and autoimmune diseases [@Videhult2016]. For example, @Azad2015 found a lower richness and increased _Enterobacteriaceae/Bacteroidetes_ ratio in infant stool to be associated with subsequent food sensitization in the population based Canadian Healthy Infant Longitudinal Development (CHILD) birth cohort study. Secondly, animal studies indicate that stress affects the gut microbiota, which in turn leads to inflammatory immune processes and even behavioral changes. For example, rodent studies showed that early life stress induced by maternal separation led to alternations of the intestinal barrier function and the microbiota [@Rea2016]. These alternations were accompanied by behavioral changes that in one study have been found to be mediated by the intestinal bacteria [@DePalma2015]. Interestingly, the administration of probiotics could prevent or ameliorate stress induced effects [@Rea2016]. In rhesus monkeys, maternal separation also altered the composition of the intestinal microbiota of the offspring by decreasing the number _Lactobacilli_ [@Bailey1999].  However, we lack studies that investigated the effect of early life stress on the development of the gut microbiota composition in humans and specifically the effect of CC on the infant gut microbiota composition has not been studied at all.  
Among the better studied factors that influence the development of the infant gut microbiota are diet and specifically breastfeeding. Breastfeeding increases the abundance of Lactobacilli and Bifidobacteria species when compared to formula fed infants [@Gregory2016; @Videhult2016]. @Yamada2017 described how oligosaccharides present in human breast milk selectively stimulate the growth of Bifidobacteria. Finally, there is evidence suggesting that breast milk contains bacteria that colonize the infant gut. For instance, @Pannaraj2017 describe that on average, 27.7% of the infant gut microbiome resembles the microbiome of the mother’s breast milk and 10.3% resembles the microbiome of the mother’s areolar skin. In contrast to early life stress, breastfeeding is known to be a health promoting factor in early life [@Hoddinott2008].  
Altogether, this research suggests that early life stress might impair the development of the intestinal microbiota with possible long term effects on physical health and behavior and that breastfeeding might be a protective factor against the negative influence of stress. In this study we investigated whether the entrance to CC leads to changes in the intestinal microbiota and whether breastfeeding plays a role in this relationship. We obtained stool samples from infants who entered CC (*n*=49) and control infants (*n*=49) before the entrance at 10 weeks of age (T0) and four weeks after the entrance (T1). We hypothesized (1) that we find no differences in the composition of the intestinal microbiota between infants who entered CC and control infants at T0 but that we find differences at at T1. The effect of CC might be moderated by a buffering effect of breastfeeding on microbiota changes. Therefore, (2) we anticipate a smaller change in microbiota composition in the breastfed group compared to the non breastfed group of infants that attended CC. 
   

# 2. Methods
## 2.1 Participants

As part of an ongoing longitudinal study (Bibo study), 220 mothers were followed since the third trimester of pregnancy, to investigate the influences of early caregiving factors on the early development of children. Uncomplicated singleton pregnancy, proficiency in the Dutch language, no drug use and the absence of physical and mental health problems were criteria for initial inclusion. Eight of the 220 women were excluded due to preterm birth or for other medical reasons. In addition, 19 mothers discontinued participation in the study during the first three postpartum months because of personal circumstances. All infants were healthy and born at full term ($\geq$ 37 weeks). Infants born by cesarean delivery (n=4) and who had used antibiotics (n=4) were excluded. Out of a total of 9 fecal samples that were available per infant, 2 were selected for this study: Ten weeks post-partum before entrance to CC (T0) and 4 weeks after the entrance (T1). If there was no sample obtained at both times points, the infant was excluded. Seven samples were excluded for technical reasons. For these samples, the Pearson correlation of $\>.98$ between duplicate bacterial profiles was not reached. The final sample included 49 infants per group. The study has been approved by the ethical Committee of the Faculty of Social Sciences, Radboud University Nijmegen (ECG / AvdK/07.563) and informed consent was obtained from each mother.



##  2.2 Procedure

### 2.2.1 Breastfeeding
Information about breastfeeding was collected through the use of diaries. The mothers received diaries towards the end of their pregnancy with instructions to take weekly notes about breastfeeding and bottle-feeding from week 1 until week 27 after birth. For each week, the average number of breast- and/or bottle- feedings per day were noted. We used the average number of breastfeedings per day until timepoint PRE and the average number of breastfeedings per day between timepoints PRE and POST as confounding variables in the analyses.

### 2.2.2 Faeces collection, DNA isolation and microbiota profiling
The parents were instructed to collect the fecal samples at home and to store them at -20°C. Samples were kept in coolers for transportation and then stored at -20°C and later at -80°C before being processed at the Microbiology Laboratory at Wageningen University. DNA isolation from faecal samples has been described in detail elsewhere [@Salonen2010]. In brief, DNA was isolated using a combination of column purification and Repeated-Bead-Beating.  Purity and concentration of DNA were assessed with a Nanodrop 1000 spectrophotometer (Thermo Fisher Scientific, Wilmington, USA). DNA was then stored at -20 C° pending microbial analysis. The analysis was then performed utilizing a previously benchmarked custom made, phylogenetic microarray, the Human Intestinal Tract Chip (HITChip) [@Rajilic-Stojanovic2009; @Claesson2009a]. The HITchip contains a duplicated set of 3,631 probes, which target the 16S rRNA gene sequences of more than 1,000 intestinal bacterial phylotypes [@Rajilic-Stojanovic2009]. After extraction of DNA, the full-length 16S rRNA gene was amplified, which was followed by in vitro transcription and labelling of the resulting RNA with Cy3/Cy5 before hybridization to the array. Each sample was hybridized at least twice to ensure reproducibility. Duplicate hybridizations with a Pearson correlation &lt;.98 were not considered for further analysis. By pre-processing the probe-level measurements with minimum–maximum normalization and Robust Probabilistic Averaging probe summarization  [@Lahti2011; @Lahti2013] the microbiota profiles were generated  into three phylogenetic levels: Order-like 16S rRNA gene sequence groups (level 1); genus-like 16S rRNA gene sequence groups with a sequence similarity &gt;90% (level 2); and phylotype-like 16S rRNA gene sequence groups with a sequence similarity &gt;98% (level 3). For our study, we focus on the genus-level variation referred to as species and relatives (‘et rel.’) and use log10-transformed signals as a proxy for bacterial abundance. 


### 2.2.3 Additional variables
Maternal age as well as infant birthweight, age in weeks at the entrance to CC and sex were obtained as potential confounding variables. Table 1 shows descriptive statistics for the variables we accounted for.

## 2.3 Statistical analysis

All analyses were performed in R version 3.5 [@R-base]. 

### 2.3.1 Data preparation and visualization
The continuous covariates "age" and "breastfeeding" were standardized to ease interpretability and convergence of hierarchical linear models. There were missing values for 5 infants for the variables "breastfeeding" and "sibling" each. The missing values were imputed using predictive mean matching as implemented in the package _mice_ [@buurenFlexibleImputationMissing2019], which resulted in 10 imputed data sets. Probes were counted in each sample to measure richness, by using an 80% quantile threshold for detection. We calculated alpha diversity (Shannon) and applied centered-log-ratio (clr) transformation [@aitchisonStatisticalAnalysisCompositional1986] to HITchip signals using the _microbiome_ package [@R-microbiome]. Euclidian distance of clr-transformed data results in Aitchiison distance [@aitchisonStatisticalAnalysisCompositional1986]. Therefore, in order to visualize the effect of CC on overall composition over time, we could apply principal component analyses to clr-transformed HITchip signals. The resulting biplots are shown in figure 1. 

### 2.3.2 Statistical models 
To determine whether and how much entrance to CC affects microbiota overall community composition, we performed a two way permutational multivariate analysis of variance (PERMANOVA) based on Aitchison distance. The PERMANOVA was performed using the function _adonis_ from the package _vegan_ [@R-vegan] with CC (CC versus HOME) interacting with time (pre versus post) as predictors controlling for breastfeeding and age. We performed differential abundance testing using Bayesian hierarchical generalized linear models with the generalized normal distribution as implemented in the _BRMS_ package [@R-brms_b]. Compared to the Gaussian distribution, the generalized Gaussian distribution has an additional parameter $\alpha$, which allows to model skewed distributions. Therefore, it can better deal with the distributional properties of clr-transformed bacterial abundances. Shrinkage was applied to the intercept and the slopes of the covariates and the predictor "Time". Regularizing priors were chosen that make the model skeptical of an effect of childcare over time. For each genus, the same model specification was used. If a model did not converge, a stronger regularizing prior was required for the standard deviation of the varying intercept (see supplementary material for exact model specification). We used the Gaussian distribution to regress alpha-diversity on the same set of predictors. We could pass the multiple imputed objects created by _mice_ directly to the fitting function in the package _brms_. Finally, we used the Random Forests (RF) algorithm implemented in the package _ranger_ [@R-ranger] to evaluate whether we can predict childcare entrance based on 130 genus abundances for both time points. RF is a tree based ensemble learning method that is well suited for classification based on microbial abundances of samples [@knightsSupervisedClassificationHuman2011]. We randomly selected 80% of the collected samples that constituted the training data set. We first tuned the RF models based on out-of-bag error. Node splitting was based on the gini criterion. Then we evaluated whether we can correctly classify CC based on 130 genus abundances using the hold out set.


# 3. Results 

```{r echo=F}
library(knitr)
opts_chunk$set(warning=F, message=F, echo=F, results="hide")
```

```{r}
library(here)
library(tidyverse)
library(papaja)
library(ggpubr)
library(microbiome)
library(ggrepel)
load(here("data/cc_analyses_workspace.RData"))
source("https://raw.githubusercontent.com/HenrikEckermann/in_use/master/reporting.R")
source("https://raw.githubusercontent.com/HenrikEckermann/in_use/master/mb_helper.R")

```


# 3. Results 

## 3.1. Microbiota composition
Thirteen genus like groups from the _Actinobacteria_, _Firmicutes_, _Proteobacteria_ and phyla showed an average abundance of $\geq 0.05$ % in at least 20% of the samples (Figure 1). Overall the microbiota was dominated by _Bifidobacterium_ with an average abundance of more than 50%. Followed by facultative anaerobic _Bacilli_ (_Streptococcus spp_, _Enterococcus_, _Lactobacilllus_ and _Granulicatella_). The general variation of relative abundance of all taxa was quite high, with _Bifidobacterium_ for instance ranging from 0.2% to 89%. 

Figures 2 and 3 show microbiota composition (Aitchison distance) for the first four principal components within the CC (A) and within the HOME (B) group. The starting points of the arrows indicate the microbiota composition in space at time-poin PRE, whereas the endpoint corresponds the composition at time-poin POST. There appear to be no differences in location between CC and HOME at PRE or POST. Also, we did not identify a clearly uniform direction of the shifts over time in either group. Figure 2 shows that within the CC group, a few infants score relatively high on PC2 and all but one move towards the center indicating shifts in the relative abundance of _Granulicatella_ and _Enterococcus_, which load high on PC2. Altogether, microbiota composition development between time points appears to be highly individual and unsystematic. 



```{r}
# # modified biplot function 
# 
# biplot <- function(
#   pseq_clr, 
#   scaling_factor = 10, 
#   color = NULL, 
#   text = FALSE, 
#   split_by = FALSE, 
#   facet = FALSE, 
#   connect_series = FALSE, 
#   subject_id = "subject_id", 
#   filter_samples = FALSE,
#   otu_color = "#ef8a62",
#   toptaxa = toptaxa
#   ) {
# 
# 
#     # PCA
#     pcx <- pseq_clr %>% 
#         otu_to_df() %>%
#         column_to_rownames("sample_id") %>%
#         prcomp()
# 
# 
#     # rename colnames 
#     rename_taxa <- function(taxa) {
#       if (taxa == "Clostridium \\(sensu stricto\\)") return("Clostridium_sensu_stricto")
#       new_taxa = gsub("\\.", "", taxa)
#       new_taxa = gsub(" ", "", new_taxa)
#       return(new_taxa)
#     }
#     # extract loadings
#     pcx_rot <- 
#         pcx$rotation %>%
#             as.tibble() %>%
#             mutate_all(function(x) x * scaling_factor) %>%
#             add_column(taxa = rownames(pcx$rotation)) %>%
#             mutate(taxa = rename_taxa(taxa))
#     pcx_rot_fit <- filter(pcx_rot, taxa %in% toptaxa)
# 
#     # combine first 4 PCs with metadata
#     princomps <- pcx$x %>% as.data.frame() %>%
#         rownames_to_column("sample_id") %>%
#         select(PC1, PC2, PC3, PC4, sample_id)
#     data <- pseq_clr %>% 
#                 sd_to_df() %>% 
#                 left_join(princomps, by = "sample_id")
# 
#     # apply filtering
#     if (filter_samples != FALSE) data <- data %>% filter(sample_id %in% filter_samples)
# 
#     # avoid errors due to wrong class
#     if (length(color) > 0) data[[color]] <-  as.factor(data[[color]])
# 
#     # if connecting by time, data must be arranged accordingly and also time/subject must be factor
#     if (connect_series != FALSE) { 
#         data[[subject_id]] <-  as.factor(data[[subject_id]])
#         data[[connect_series]] <-  as.factor(data[[connect_series]])
#         data <- data %>% arrange_(subject_id, connect_series)
#     } 
# 
# 
# 
# 
# 
# 
#     # how much variance do pcs explain?
#     pc1 <- round(pcx$sdev[1]^2/sum(pcx$sdev^2),3)
#     pc2 <- round(pcx$sdev[2]^2/sum(pcx$sdev^2),3)
#     pc3 <- round(pcx$sdev[3]^2/sum(pcx$sdev^2),3)
#     pc4 <- round(pcx$sdev[4]^2/sum(pcx$sdev^2),3)
# 
# 
#     # define plottting function 
#     create_plot <- function(data, pc = 1, pc1, pc2, title = "") {
#         data %>%        
#         ggplot(aes_string(glue("PC{pc}"), glue("PC{pc+1}"), label = "sample_id", color = color)) +
#             geom_text_repel(data = pcx_rot_fit, aes_string(glue("PC{pc}"), glue("PC{pc+1}"), label = "taxa"), color = otu_color, size = 9) +
#             xlab(glue("PC{pc}: [{pc1*100}%]")) +  ylab(glue("PC{pc+1}: [{pc2*100}%]")) +
#             scale_y_continuous(sec.axis = ~./scaling_factor) +
#             scale_x_continuous(sec.axis = ~./scaling_factor) +
#             scale_color_manual(values = c("#fc8d62", "#8da0cb", "#66c2a5",'#1f78b4','#33a02c','#e31a1c')) +
#             ggtitle(title) +
#             theme_bw(base_size = 20)  
#     }
# 
# 
#     # split by (to produce bigger plots than possible just by facet_wrap or to use in addition as grouping possibility)
#     if (split_by != FALSE) {
#         data <- data %>% group_by_(split_by) %>% nest()
#         pc_plots_1 <- map2(data[[2]], data[[1]], ~create_plot(data = .x, title = .y, pc = 1, pc1, pc2))
#         pc_plots_2 <- map2(data[[2]], data[[1]], ~create_plot(data = .x, title = .y, pc = 3, pc3, pc4))
#         pc_plots <- append(pc_plots_1, pc_plots_2)
#     } else {
#         # plots
#         p12 <- create_plot(data = data, pc = 1, pc1, pc2)
#         p34 <- create_plot(data = data, pc = 3, pc3, pc4)
#         pc_plots <- list(p12, p34)  
#     }
# 
# 
# 
#     # apply optionals 
#     # text 
#     if (text) {
#         pc_plots <- map(pc_plots, ~.x + geom_text(size = 8))
#     }else{
#         pc_plots <- map(pc_plots, ~.x + geom_point(size = 5))
#     }
# 
# 
#     # path 
#     if (connect_series != FALSE) {
#       pc_plots <- map(pc_plots, ~.x + geom_path(aes_string(group = subject_id), arrow = arrow(length = unit(0.8,"cm"), ends = "last"), alpha = 0.3, size = 1.8))
# 
# 
#     } 
# 
#     # facetting 
#     if (facet != FALSE) pc_plots <- map(pc_plots, ~.x + facet_wrap(as.formula(glue(".~{facet}"))))  
# 
#     pc_plots
# }
```



```{r}
# define which taxa will be printed
toptaxa <- comp_all_df %>%
    filter(prob < 0.05 | prob > 0.95, abs(mean) >= 0.18) %>%
    select(-prob) %>%
    arrange(desc(abs(mean)))
```


```{r fig.width=30, fig.height = 15, fig.show=T, fig.cap='Development of microbiota composition over time within CC (A and C) and HOME (B and D).'}
# biplot_time_2 <- biplot(pseq.clr, split = "cc", connect_series = "time", otu_color = "#ca0020", toptaxa = toptaxa$genus)
# ggarrange(
#     biplot_cc[[1]] + xlim(-12.5, 5) + ggtitle('A'), 
#     biplot_cc[[2]] + xlim(-12.5, 5) + ggtitle('B'), 
#     biplot_cc[[3]] + xlim(-12.5, 5) + ggtitle('C'), 
#     biplot_cc[[4]] + xlim(-12.5, 5) + ggtitle('D'), 
#     nrow = 2, ncol = 2, 
#     common.legend = T)

scaling_factor <- 10
library(patchwork)

# (biplot_time_2[[1]] + 
#   scale_y_continuous(limits = c(-5.25, 8), sec.axis = ~./scaling_factor) +
#   scale_x_continuous(limits = c(-12.5, 5), sec.axis = ~./scaling_factor) + ggtitle('A1') |
#   biplot_time_2[[2]] +
#   scale_y_continuous(limits = c(-5.25, 8), sec.axis = ~./scaling_factor) +
#   scale_x_continuous(limits = c(-12.5, 5), sec.axis = ~./scaling_factor) + ggtitle('B1')) / 
#   (biplot_time_2[[3]] + 
#   scale_y_continuous(limits = c(-5.5, 7), sec.axis = ~./scaling_factor) +
#   scale_x_continuous(limits = c(-6.25, 5), sec.axis = ~./scaling_factor) + ggtitle('A2') |
#   biplot_time_2[[4]] +
#   scale_y_continuous(limits = c(-5.5, 7), sec.axis = ~./scaling_factor) +
#   scale_x_continuous(limits = c(-6.25, 5), sec.axis = ~./scaling_factor) + ggtitle('B2')) + plot_layout() &theme_bw(base_size =40)
```


```{r fig.width=30, fig.height = 15, fig.show=T, fig.cap='First two principal components illustrating development of microbiota composition over time within CC (A) and HOME (B).'}
# biplot_time_2[[1]] + 
#   scale_y_continuous(limits = c(-5.25, 8), sec.axis = ~./scaling_factor) +
#   scale_x_continuous(limits = c(-12.5, 5), sec.axis = ~./scaling_factor) + ggtitle('A') +
#   biplot_time_2[[2]] +
#   scale_y_continuous(limits = c(-5.25, 8), sec.axis = ~./scaling_factor) +
#   scale_x_continuous(limits = c(-12.5, 5), sec.axis = ~./scaling_factor) + ggtitle('B') + plot_layout(ncol = 2) & theme_bw(base_size = 40)
```

```{r fig.width=30, fig.height = 15, fig.show=T, fig.cap='Third and fourth principal component illustrating development of microbiota composition over time within CC (A) and HOME (B).'}
# biplot_time_2[[3]] + 
#   scale_y_continuous(limits = c(-5.5, 7), sec.axis = ~./scaling_factor) +
#   scale_x_continuous(limits = c(-6.25, 5), sec.axis = ~./scaling_factor) + ggtitle('A') +
#   biplot_time_2[[4]] +
#   scale_y_continuous(limits = c(-5.5, 7), sec.axis = ~./scaling_factor) +
#   scale_x_continuous(limits = c(-6.25, 5), sec.axis = ~./scaling_factor) + ggtitle('B') + plot_layout(ncol = 2) & theme_bw(base_size = 40)
```





```{r fig.width=30, fig.height = 15, fig.show=T, fig.cap='First two principal components illustrating development of microbiota composition over time within CC (A) and HOME (B).'}
biplot_siblings <- biplot(pseq_plot, connect_series = "time",  split_by = "sibling",  color = "csection", loading = 0.32, text = F, otu_text_size = 8, otu_color = "#bababa", size = 8, arrow_size = 1, path_size = 0.75, alpha = 1, otu_alpha = 0.75, path_alpha = 0.3, colors = c("#404040", "#ca0020"))
biplot_siblings <- map(biplot_siblings, ~.x + theme_bw(base_size = 40) + theme(legend.position="none"))
biplot_siblings[[1]] + 
  scale_y_continuous(limits = c(-5.25, 8), sec.axis = ~./scaling_factor) +
  scale_x_continuous(limits = c(-12.5, 5), sec.axis = ~./scaling_factor) + ggtitle('A') +
  biplot_siblings[[2]] +
  scale_y_continuous(limits = c(-5.25, 8), sec.axis = ~./scaling_factor) +
  scale_x_continuous(limits = c(-12.5, 5), sec.axis = ~./scaling_factor) + ggtitle('B') + plot_layout(ncol = 2)
```



```{r fig.width=30, fig.height = 15, fig.show=T, fig.cap='First two principal components illustrating development of microbiota composition over time for infants with or without siblings. .'}
biplot_siblings[[4]] + 
  scale_y_continuous(limits = c(-5.25, 7.5), sec.axis = ~./scaling_factor) +
  scale_x_continuous(limits = c(-6.5, 5), sec.axis = ~./scaling_factor) + ggtitle('A') +
  biplot_siblings[[5]] +
  scale_y_continuous(limits = c(-5.25, 7.5), sec.axis = ~./scaling_factor) +
  scale_x_continuous(limits = c(-6.5, 5), sec.axis = ~./scaling_factor) + ggtitle('B') + plot_layout(ncol = 2)
```

## 3.2 Effects of CC on microbiota composition

Table 1 shows demographic variables for both groups. There was a significant difference in age between groups at PRE (_t_(62.43) = -4.54, _p_ < .001) and at POST (_t_(60.82) = -4.88, _p_ < .001) according to Welch's t-test. Besides that, there were no differences between groups for any of the remaining variables. To account for the differences in age and the naturally varying amounts of breastfeeding, we used both variables as covariates in all linear models.

```{r results='asis'}
# load tables 
source(here("Rmd/article/tables/apa_tables_docx.R"))
apa_table(list_of_demographics, escape = F, caption = table_caption_descr, note = table_note_descr)
```

### 3.2.1 Permutational multivariate ANOVA 
We compared the overall community composition using PERMANOVA based on Aitchison distance metric. An assumption for PERMANOVA is multivariate homogeneity of group dispersions (variances) [@andersonPermutationalMultivariateAnalysis2017]. We used the function _betadisper_ [@R-vegan], which utilizes the _PERMDISP2_ procedure as implemented by Marti Anderson and found that this assumption was met for the factors "CC" `r report_f(hg_cc[1, 4], hg_cc[1, 1], hg_cc[2, 1], hg_cc[1, 5])`, "Time" `r report_f(hg_time[1, 4], hg_time[1, 1], hg_time[2, 1], hg_time[1, 5])`,
"Csection" `r report_f(hg_csection[1, 4], hg_csection[1, 1], hg_csection[2, 1], hg_csection[1, 5])` and the subgroups that result out of the interaction of "Time" and "CC" `r report_f(hg_groups[1, 4], hg_groups[1, 1], hg_groups[2, 1], hg_groups[1, 5])`. However, the dispersions between the groups of "Sibling" were not homogeneous `r report_f(hg_sibling[1, 4], hg_sibling[1, 1], hg_sibling[2, 1], hg_sibling[1, 5])` and therefore the resulting test statistics should be interpreted with caution. We did not find a significant effect of CC over time on overall community composition (see table 2). Breastfeeding, sibling, c-section and age significantly predicted overall community composition. Figure x shows the genera that mostly changed as a function of each significant predictor. Despite the low number of infants born by cesarian section, PERMANOVA indicates that cesarean section significantly influences gut microbiota composition with the strongest effect on Bifidobacteria.


```{r results="asis"}
rename_coefs <- c("Time", "CC", "Age", "Breastfeeding", "C-section", "Sibling", "Birthweight", "Time x CC", "Residuals", "Total")
pm_table <- pm_table %>% mutate("Model Parameter" = rename_coefs)
apa_table(pm_table, caption = "Model Output PERMANOVA", note = NULL)
#pm_table
```


```{r fig.width=15, fig.height = 7.5,fig.show=T, fig.cap='Top Taxa that differ most as a function of each predictor.'}
library(patchwork)
(pmps[[1]] + ggtitle('A')  | pmps[[2]] + ggtitle('B')) /
  (pmps[[3]] + ggtitle('C')  | pmps[[4]] + ggtitle('D'))  +plot_layout() & theme_bw(base_size = 18)  
```

```{r fig.width=15, fig.height = 7.5,fig.show=T, fig.cap='Top Taxa that differ most as a function of each predictor.'}
(pmps[[5]] + ggtitle('E')  | pmps[[6]] + ggtitle('F')) /  
 (pmps[[7]] + ggtitle('G')  | pmps[[8]] + ggtitle('H')) +plot_layout() & theme_bw(base_size = 18)
```

#### 3.2.2 Differential abundance testing (hierarchical GLM)

We modeled clr-transformed bacterial abundance using the generalized Gaussian distribution with constant variance $\sigma$ and skewness parameter $\alpha$. The parameter $\mu$ is modeled as a linear function of the predictors "CC", "Time", "breastfeeding", "sibling", "csection" and "age". Figures x-x show the arithmetric means of the posterior distribution of the differences in $\mu$ between (figure x) or within (figure x) the groups with 95% highest probability density interval. We show only those genera that were different with high probability ($\geq 95$ %) in at least one comparison or predictor. Holding all other predictors constant, the GLM predicts a relative increase in the abundance of _Streptococcus mitis et rel_, _Streptococcus intermedius et rel_, _Granulicatella_ and _Enterococcus_ with increasing age. Breastfeeding is positively associated with _Streptococcus mitis et rel_ and _Staphylococcus_ and negatively associated with _Granulicatella_, _Collinsella_ and _Enterococcus_. Comparing CC to HOME, we find no clear difference at time-point PRE. At time-point POST we find a relative decrease in _Streptococcus intermedius et rel_. We find a stronger decrease of _Streptococcus mitis et rel_, _Streptococcus intermedius et rel_ and _Granulicatella_ in the CC group compared to HOME. These findings are mostly in line with the previously presented analyses and biplots. Since there is a low number number of c-sections in our group (9), the posterior distribution of the group difference between cesction is expected to be wide reflecting high uncertainty. Furthermore, the skeptical prior assigns highest probability to the effect being equal to zero. Nevertheless, the model detects a clear decrease in _Bifidobacterium_ for infants born by c-section, which is less strong for those infants with siblings.


```{r fig.width=10, fig.height = 10, fig.show=T, fig.cap='Posterior distribution of the slopes of age and breastfeeding. Red color indicated that 95% of the posterior distribution is greater or smaller than zero.'}
comparison_plots_article_3
```

```{r fig.width=10, fig.height = 10, fig.show=T, fig.cap='Posterior distribution of the difference in mu between CC and HOME. Red color indicated that 95% of the posterior distribution is greater or smaller than zero.'}
comparison_plots_article_1
```



```{r fig.width=10, fig.height = 10, fig.show=T, fig.cap='Posterior distribution of the difference in mu within CC and HOME. Red color indicated that 95% of the posterior distribution is greater or smaller than zero.'}
comparison_plots_article_2
```


```{r fig.width=10, fig.height = 10, fig.show=T, fig.cap='Posterior distribution of the difference in mu between sibling and delivery mode. Red color indicated that 95% of the posterior distribution is greater or smaller than zero.'}
comparison_plots_article_4
```


```{r fig.width=30, fig.height = 10, fig.show=T, fig.cap='Individual paths between PRE and POST samples for Streoptococcus mitis et rel (A) and Streptococcus bovis et rel (B).'}
ggarrange(abundance_paths[[1]] + ggtitle('A'), 
  abundance_paths[[2]] + ggtitle('B')) 
```

```{r fig.width=30, fig.height = 10, fig.show=T, fig.cap='Individual paths between PRE and POST samples for Streoptococcus intermedius et rel (A) and Granulicatella (B).'}
ggarrange(abundance_paths[[3]] + ggtitle('A'), 
  abundance_paths[[4]] + ggtitle('B')) 

```




```{r fig.width=15, fig.height = 15, fig.show=T, fig.cap='Individual paths between PRE and POST samples for Bifidobacterium for infants with or without siblings. Infants born via cesearean section are indicated by red color.'}
csec_sib
```


#### 3.2.3 Alpha diversity (hierarchical LM)
 

Alpha-diversity was assumed to be Gaussian distributed with constant variance $\sigma$. Table x shows the estimated difference in the parameter $\mu$ of the generalized Gaussian distribution between groups as well as the estimated $\sigma$. There was no difference in alpha diversity within the HOME group or between HOME and CC before childcare entrance. However, comparing $\mu$ within CC or between HOME and CC after entrance, we see that diversity is lower in the CC group with high certainty suggesting that CC lead to a decrease in alpha diversity. Figure 3 shows alpha diversity for each subject for each subgroup. It furthermore shows the highest probability density interval of $\mu$ for each group. We see that $\mu$ was highest in the CC group before entrance and lowest of all groups after CC attendance. However, we see again large individual variation within each group. Besides that, cesarian section leads to an increase in alpha-diversity, whereas having a sibling is associated with a decreased alpha-diversity.

```{r results="asis"}
apa_table(
  comp_comparisons,
  caption = "Estimated model parameters alpha diversity.",
  added_stub_head = "Variables"
)
```




```{r fig.width=10, fig.height = 10, fig.show=T, fig.cap='Observed values of Shannon alpha-diversity (points), individual paths and 95% highest probability density interval of the posterior distribution of mu when age and breastfeeding are kept at sample average.'}
p_comp + theme_bw(base_size = 20)
```



## 3.4 Random Forest 

According to our hypotheses, we would expect to be able to classify whether an infant belongs to the CC group at time-point POST. In contrast, at time-point PRE we would expect prediction accuracy to be lower since there should be no differences between CC and HOME. Neither the PRE model, nor the POST model achieved a higher prediction accuracy than 0.51 suggesting that there was no systematic effect of childcare entrance on microbiota composition. 
 



# 4. Discussion 

We utilized both multivariate and univariate statistical approaches to test our hypotheses that CC has an effect on gut microbiota composition . Our analyses did not indicate a significant effect of CC over time and therefore both null hypotheses could not be rejected for our sample. Unexpectedly, breastfeeding and CC together explained only small amounts of variation in overall microbiota composition.  
This result is in contrast to the previously cited rodent studies [@Rea2016], but in line with @Falony2016 who recently explored the drivers of microbiota variation and similarly showed very small effect sizes for all covariates under investigation. In total, 69 of 503 (13.7%) metadata variables were significantly associated with microbiota composition and these had a cumulative (non-redundant) effect size on community variation of only 7.63%. Thus, more than 92% of microbiota variation remained unexplained suggesting additional contribution from unknown factors, such as stochastic effects and biotic interactions.  
In animal experiments the variable under investigation is usually the only factor that is changed, while in human populations the hosts show much more variation with regards to genetics and lifestyle, while the microbiota is subject to much more interactions with the environment [@Falony2016; @Rea2016].  
Nevertheless, the structure, function and stability of the infant microbiota is significantly different in composition as well as diversity from that of adults [@DeMuinck2018]. Especially in the first few weeks the gut microbiota composition is highly unstable and individual. This period is followed up by a more stable phase dominated by _Bifidobacterium_ species when breastfed and gradually goes to a more mature and diverse community during weaning due to the introduction of solid foods. It is yet unclear at what age the microbiota fully matures, but estimates range from 1 to 3 years of age [@DeMuinck2018].
This difference in microbiota dynamics in the first months of life could explain the small effect sizes we observed. Because the microbiota is so unstable it might be that there are no general effects of microbiota composition, in other words: Every infant responds in a unique way. This seems to be the case as can be seen by figures 5-14 where we do observe an average response, but a high percentage of infants shows an opposite effect. Inspecting the same figures for the remaining 120 bacterial groups, we also find highly variable trends between T0 and T1, which explains why we don’t observe an overall general effect. Another line of reasoning could be that, because of the lower stability of the infant microbiota compared to adults, it is therefore more easily ‘pushed’ or shifted due to an environmental influence such as the entrance to CC. This could still be the case. However, our data suggests the effects of this ‘push’ then are highly individual.  
Another observation is that unfortunately the period of the introduction of the first solid foods often coincided with the period under investigation, possibly combining multiple additional variables that affect the microbiota, thus obscuring separate effects. Further research would elucidate whether this is the case and we could then make classes for instance based on the percentage of solid food.  
Contrasting with previous studies we found no significant differences in the abundance of _Bifidobacterium_ between breastfed and non-breastfed infants (see figure 15). This can be explained by the addition of prebiotic fibers in infant formula’s such as Galactooligosaccharides (GOS) and Fructooligosaccharides (FOS) [@Borewicz2018]. These fibers have been added in 2002 to most infant formulae (personal communication) to emulate the bifidogenic effect of breastmilk.  
In addition to biological explanations for our observations, we encountered several methodological limitations. First, only a limited number of infants were not breastfed per group (18 infants who entered childcare and 15 infants who did not enter childcare). Larger sample sizes might be required to find an effect of breastfeeding or interactions between breastfeeding and CC. Secondly, the distributional properties of the taxa could not be addressed by solely using a gaussian model. Despite log transforming the abundances, the assumptions of normality of residuals and heterogeneity of variance often could not be met for our LMEs. We used non-parametric tests for exploratory purposes, which are not affected by the distributional properties but these do not take into account individual variation, which is a strength of LME when testing repeated measurements [@Barr2013]. A possible solution could be to analyze the data taking into consideration the individual distribution for each bacterial genus e.g. by using different link functions for the generalized linear mixed effects models. It should be noted though that for the purpose of inference as opposed to prediction, the assumptions of normality of residuals and homogeneity of variance are less important and at least mild violations should be unproblematic [@Gelman2007].  
In conclusion, based on our analyses it seems that childcare attendance does not influence the microbiota in a large and distinct way, beyond normal temporal variation. This does not mean that childcare attendance does not have an effect on the microbiota, but might imply that its effect on the microbiota are either very individual or are only observable at a different level, such as through bacterial metabolites.

<!--  -->